src/declarations/custom.d.ts

```diff
@@ -2,3 +2,8 @@ declare module "*.svg" {
   const content: string;
   export default content;
 }
+
```

Comment:
I am not sure if we exported DownloadQRCode.png from figma correctly, there shouldn't be a bottom margin as we seen in the preview.
`Rectangle 3214` is the correct one I guess 🤔